CMakeLists.txt

```diff
@@ -187,7 +187,11 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(IS_PYTHON_BUILD)
-    find_package(PythonLibs REQUIRED)
+    if(WIN32)
```

Comment:
We should use a variable to set which version of python to check for, then do it the same way across all platforms